### DISCUSSION TITLE
Github Pages Date Issue

### DISCUSSION THREAD

**Discussion by angelabaltes** [community/community#22675] [Other Feature Feedback, Questions, & Ideas]
<p>I have added a new tab to my website called" teaching", and I have noticed that the date field is not showing up as it does on other pages. It looks like this( , , 2020). I have added it to navigation.yml, _config.yml and created a teaching.html in _pages.</p>
<p>Could it be I have failed to configure something?</p>
<p>*My repo is: <a href="https://github.com/angelabaltes/angelabaltes.github.io" rel="nofollow noopener">https://github.com/angelabaltes/angelabaltes.github.io</a></p>
<p>*and my github pages site is: <a href="https://angelabaltes.com/teaching/" rel="nofollow noopener">https://angelabaltes.com/teaching/</a></p>
<p>I would most appreciate your help. It has to be something minor that I have overlooked.</p>

**Comment by thinkverse** [comment, marked as answer]
<p>Hello. :wave:</p>
<p>I forked the site and experimented a bit, I found the issue is in <code>_includes/archive-single.html</code> at <code>line 42</code> when its formatting the year, it’s expecting a <code>post.type</code> and <code>post.venue</code> which aren’t in the <code>.md</code> files, causing only <code>, , 2020</code> to be displayed, to just display the date you can replace the line with this:</p>
<pre><code class="lang-auto">&lt;p&gt; &lt;time datetime="{{ post.date | default: "1900-01-01" | date_to_xmlschema }}"&gt;{{ post.date | default: "1900-01-01" | date: "%B %d, %Y" }}&lt;/time&gt; &lt;/p&gt;
</code></pre>
<p>Which is a piece taken from <code>line 46</code> in the same file (<code>_includes/archive-single.html</code>) - displaying the month, day and year, or you could add the <code>post.type</code> and <code>post.venue</code> to the <code>.md</code> file meta.</p>
<p>I hope that helps you. :slightly_smiling_face:</p>

**Comment by angelabaltes** [comment]
<p>This was incredibly helpful! Thank you!</p>
